--- gnusound-0.7.5/configure.ac.orig	2008-07-06 10:03:19.000000000 +0200
+++ gnusound-0.7.5/configure.ac	2023-09-23 20:34:32.123550409 +0200
@@ -266,7 +266,7 @@ AC_ARG_WITH(gnome2,
 if test "x$use_gnome2" = "xyes"; then
   PKG_CHECK_MODULES(GNOME2,libgnomeui-2.0 >= 2.0.0 libglade-2.0 gthread-2.0,
         AC_DEFINE(HAVE_GNOME2,,[use GNOME2])
-        CFLAGS="$CFLAGS -DGNOME_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -rdynamic $GNOME2_CFLAGS"
+        CFLAGS="$CFLAGS -rdynamic $GNOME2_CFLAGS"
         LIBS="$LIBS -rdynamic $GNOME2_LIBS",
         use_gnome2=no)
 fi
--- gnusound-0.7.5/src/gui.c.orig	2005-06-19 17:44:26.000000000 +0200
+++ gnusound-0.7.5/src/gui.c	2023-09-23 20:53:53.150593919 +0200
@@ -23,9 +23,6 @@
  * Grab bag of GUI initialization and related utility functions.
  */
 
-#define GTK_DISABLE_DEPRECATED 1
-#define GNOME_DISABLE_DEPRECATED 1
-
 /* Has to come _before_ config.h ... */
 
 #include <glade/glade.h>
--- gnusound-0.7.5/src/gui.h.orig	2005-06-19 17:44:26.000000000 +0200
+++ gnusound-0.7.5/src/gui.h	2023-09-23 20:56:56.786265746 +0200
@@ -24,8 +24,6 @@
 #include <config.h>
 
 #ifdef HAVE_GNOME2
-# define GTK_DISABLE_DEPRECATED   1
-# define GNOME_DISABLE_DEPRECATED 1
 # define GUI_GTK_VERSION_TAG      "-2"
 #else
 # define GUI_GTK_VERSION_TAG      ""
