--- gnusound-0.7.5/doc/Makefile.in.orig	2023-09-23 18:52:44.919969304 +0200
+++ gnusound-0.7.5/doc/Makefile.in	2023-09-23 19:04:00.322976999 +0200
@@ -24,8 +24,7 @@ docname=$(name)
 xslstylesheet=@DB2XHTML_STYLESHEET@
 omfdir=@datadir@/omf/$(name)
 
-owner_user ?= root
-owner_group ?= root
+DESTDIR ?=
 
 dist:
 	for lang in $(LANGUAGES); do \
@@ -47,18 +46,18 @@ distclean: veryclean
 	rm -f Makefile
 
 uninstall:
-	rm -rf $(docrootdir)
+	rm -rf $(DESTDIR)$(docrootdir)
 
 install:
-	mkdir -p $(omfdir)
+	mkdir -p $(DESTDIR)$(omfdir)
 	for lang in $(LANGUAGES); do \
-		mkdir -p $(docrootdir)/$$lang; \
-		install -m 644 -o $(owner_user) -g $(owner_group) \
+		mkdir -p $(DESTDIR)$(docrootdir)/$$lang; \
+		install -m 644 \
 			$$lang/*.xml $$lang/*.html $$lang/topic.dat \
-			$(docrootdir)/$$lang; \
-                install -m 644 -o $(owner_user) -g $(owner_group) \
+			$(DESTDIR)$(docrootdir)/$$lang; \
+                install -m 644 \
                         $$lang/$(name).omf \
-                        $(omfdir)/$(name)-$$lang.omf; \
+                        $(DESTDIR)$(omfdir)/$(name)-$$lang.omf; \
 	done
 
 .PHONY: dist clean veryclean distclean install uninstall
--- gnusound-0.7.5/Makefile.in.orig	2008-07-06 10:14:12.000000000 +0200
+++ gnusound-0.7.5/Makefile.in	2023-09-23 21:04:26.860494151 +0200
@@ -22,8 +22,10 @@ export CFLAGS=@CFLAGS@ \
 	-DSYSCONFDIR="${sysconfdir}" \
 	-DPREFIX="${prefix}" \
 	-DDATADIR="${datadir}" 
-export LDFLAGS=@LIBS@ @LDFLAGS@
+export LDFLAGS=@LDFLAGS@ @LIBS@
 
+DESTDIR ?=
+
 name=@PACKAGE@
 version=@VERSION@
 prefix=@prefix@
@@ -33,9 +35,6 @@ bindir=@bindir@
 libdir=@libdir@/$(name)
 datadir=@datadir@
 
-owner_user ?= root
-owner_group ?= root
-
 gnusounddir=@GNUSOUNDDIR@
 guidir=$(libdir)/gui
 docdir=$(datadir)/gnome/help/$(name)
@@ -190,17 +189,17 @@ $(objects): %.o: %.c src/config.h src/de
 
 src/$(name): $(objects) .ldflags
 	rm -f src/$(name)
-	$(CC) `cat .ldflags` $(objects) -o src/$(name)
+	$(CC) $(objects) -o src/$(name) `cat .ldflags` -lm
 
 install: all 
-	$(MAKE) -C doc install owner_user=$(owner_user) owner_group=$(owner_group)
-	$(MAKE) -C modules install owner_user=$(owner_user) owner_group=$(owner_group)
-	mkdir -p $(bindir)
-	mkdir -p $(guidir)
-	mkdir -p $(desktopdir)
-	install -m 755 -o $(owner_user) -g $(owner_group) src/$(name) $(bindir)
-	install -m 644 -o $(owner_user) -g $(owner_group) $(gui_files) $(guidir)
-	install -m 644 -o $(owner_user) -g $(owner_group) $(desktop_files) $(desktopdir)
+	$(MAKE) -C doc install 
+	$(MAKE) -C modules install
+	mkdir -p $(DESTDIR)$(bindir)
+	mkdir -p $(DESTDIR)$(guidir)
+	mkdir -p $(DESTDIR)$(desktopdir)
+	install -m 755 src/$(name) $(DESTDIR)$(bindir)
+	install -m 644 $(gui_files) $(DESTDIR)$(guidir)
+	install -m 644 $(desktop_files) $(DESTDIR)$(desktopdir)
 
 uninstall:
 	$(MAKE) -C modules uninstall
@@ -251,7 +250,7 @@ distcheck: dist
 		cd $(name)-$(version) && \
 		./configure --prefix=`pwd`/../install CFLAGS= && \
 		make && \
-		make install owner_user=`whoami` owner_group=`whoami`)
+		make install)
 
 maintainer-install: dist
 	tar xjf $(name)-$(version).tar.bz2 
diff -Nurb gnusound-0.7.1.bef/modules/Makefile.in gnusound-0.7.1.new/modules/Makefile.in
--- gnusound-0.7.1.bef/modules/Makefile.in	2005-02-13 05:27:57.000000000 +0100
+++ gnusound-0.7.1.new/modules/Makefile.in	2005-05-31 01:40:18.000000000 +0200
@@ -28,8 +28,7 @@
 moduledir=$(libdir)/modules
 gnusounddir=@GNUSOUNDDIR@
 
-owner_user ?= root
-owner_group ?= root
+DESTDIR ?= 
 
 src=$(shell ls *.c)
 obj=$(addsuffix .so, $(basename $(src)))
@@ -60,8 +59,8 @@
 	$(CC) `cat $(gnusounddir)/.cflags` -I$(gnusounddir)/src -I$(gnusounddir)/src/$(gtk) -fPIC -shared  $< -o $@  `cat .$(basename $@).cflags`
 
 install:
-	mkdir -p $(moduledir)
-	install -m 644 -o $(owner_user) -g $(owner_group) $(module_files) $(moduledir)
+	mkdir -p $(DESTDIR)$(moduledir)
+	install -m 644 $(module_files) $(DESTDIR)$(moduledir)
 
 uninstall:
 	(cd $(moduledir) && rm -f $(module_files))
--- ../BUILD/gnusound-0.7.5/Makefile.in.orig	2023-09-23 21:05:55.943344881 +0200
+++ ../BUILD/gnusound-0.7.5/Makefile.in	2023-09-23 21:10:40.288471118 +0200
